src/main.py

```diff
@@ -354,7 +386,8 @@ def analyze_line(line: str) -> str:
             full_msg = [match.group(1), match.group(2)]
             if (full_msg[0] == players[0].name or
                     full_msg[0] == players[1].name):
-                full_msg[0] = '☆' + full_msg[0]
+                #full_msg[0] = '☆' + full_msg[0]
+                full_msg[0] = '*' + full_msg[0]
```

Comment:
I'm currently using the star character that Showdown already uses (indicating a player who is battling is speaking) instead of the asterisk (which indicates a bot on Showdown).